### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by mattam82**
**Kind:** documentation 

As discussed during the last Coq call, point to the wiki in INSTALL.md for more general install instruction (and pointers too known issues, as the one recently found by PMP on Debian.)

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Installing From Sources
 =======================
 
+This document presents instructions to install this branch of Coq.
+For more general installation instructions and information about know
+build system issue, please consult the wiki page:
```
Comment (Zimmi48) [inline INSTALL.md:6]:
```suggestion
build system issues, please consult the wiki page: 
```

**Comment by Zimmi48** [comment]
Be careful that, as someone with write-access, you *cannot* use the web edit feature of GitHub because it doesn't support creating a branch in your fork. In this case, you accidentally created a branch in the main repository instead. Let's merge this PR quickly so that we can delete the branch.

File: INSTALL.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 Installing From Sources
 =======================
 
+This document presents instructions to install this branch of Coq.
+For more general installation instructions and information about know
```
Comment (jfehrle) [inline INSTALL.md:5]:
```suggestion
For more general installation instructions and information about known
```

**Review by Zimmi48 — APPROVED** [review]
@coqbot: merge now

**Pull request merged**